pkgs/by-name/si/sirula/package.nix

```
@@ -1,21 +1,21 @@
 {
+  nix-update-script,
   lib,
   fetchFromGitHub,
   rustPlatform,
   pkg-config,
   gtk3,
   gtk-layer-shell,
 }:
-
-rustPlatform.buildRustPackage rec {
+rustPlatform.buildRustPackage {
   pname = "sirula";
-  version = "1.0.0-unstable-2023-09-02";
+  version = "1.1.0";
 
   src = fetchFromGitHub {
     owner = "DorianRudolph";
-    repo = pname;
-    rev = "b15efe85ef1fe50849a33e5919d53d05f4f66090";
-    hash = "sha256-S0WbqY49nKaBUMWfgDKZxFLJuk7uFcnTfV8s86V0Zxs=";
+    repo = "sirula";
+    rev = "f4da8f5fe50c367e7684334c31ec65fc0fe6997c";
```

```suggestion
    tag = "v${version}";
```

Why can't this be a tagged version? There is a tag in [the main repo](https://github.com/DorianRudolph/sirula/tags)

---

+1, change this to tag.